Annex60/Fluid/Movers/Examples/MoverParameter.mo

```diff
@@ -54,12 +54,13 @@ model MoverParameter
     redeclare package Medium = Medium,
     dynamicBalance=false,
     filteredSpeed=false,
-    speeds={0,1000,2000},
-    redeclare Annex60.Fluid.Movers.Data.Pumps.Wilo.Stratos25slash1to4 per,
-    inputType=Annex60.Fluid.Types.InputType.Constant,
-    speed=2000) "Pump with speed input"
+    redeclare Annex60.Fluid.Movers.Data.Pumps.Wilo.Stratos25slash1to4 per(
+      speeds_rpm={0,1000,2000},
+      constantSpeed_rpm=2000),
```

Comment:
I think this causes the different unit test results you mention in #397 . The speed for which the pressure curve is defined is 1800 rpm. By changing `constantSpeed_rpm` you also change the speed for which the curve is defined, which it should not do. As illustrated by this problem I still think it is a good idea to set `speed_rpm_nominal` in the record itself, instead of assigning it the value of `speeds_rpm[size(speeds_rpm, 1)]`?